rule/var_naming.go

```diff
@@ -22,12 +22,26 @@ var knownNameExceptions = map[string]bool{
 	"kWh":          true,
 }
 
+// meaninglessPackageNames is the list of "bad" package names from https://go.dev/wiki/CodeReviewComments#package-names
+// and https://go.dev/blog/package-names#bad-package-names.
+// The rule warns about the usage of any package name in this list if skipPackageNameChecks is false.
+// Values in the list should be lowercased.
+var meaninglessPackageNames = map[string]struct{}{
+	"common":     {},
+	"interfaces": {},
+	"misc":       {},
+	"types":      {},
+	"util":       {},
+	"utils":      {},
+}
+
 // VarNamingRule lints the name of a variable.
 type VarNamingRule struct {
 	allowList             []string
 	blockList             []string
-	allowUpperCaseConst   bool // if true - allows to use UPPER_SOME_NAMES for constants
-	skipPackageNameChecks bool
+	allowUpperCaseConst   bool                // if true - allows to use UPPER_SOME_NAMES for constants
+	skipPackageNameChecks bool                // check for meaningless and user-defined bad package names
+	badPackageNames       map[string]struct{} // inactive if skipPackageNameChecks is false
```

Comment:
Also, one point, these are appended to the one that are mentioned in the Go documentation.

Meaning, you cannot remove "common" from the list, by using this list.

I'm fine with this, but I would like you to consider two alternative options
- rename badPackageNames to something like extraWhateverPackageNames
- consider that the boolean is only about deactivating the list provided by golang

I feel like it could be clearer. If someone puts something in the exclusions, they wants it, and the boolean is only there for the hardcoded ones